packages/manager/src/config/orchestratorReq.ts

```diff
@@ -13,3 +14,23 @@ export const orchestratorReq = axios.create({
 export const setOrchestratorAuthorizationHeader = (token: string) => {
   orchestratorReq.defaults.headers.common.Authorization = `Bearer ${token}`;
 };
+
+orchestratorReq.interceptors.response.use(
```

Comment:
Can you please add a `TODO(akhilmhdh): change this to static function and persistence, will be done by akhilmhdh `

I'll raise a PR for this change no worries. Just add this comment so that I can later check out

Reply:
added